pkg/server/deploy/deploy.go

```diff
@@ -28,8 +28,7 @@ type Operations interface {
 type K8sOperations interface {
 	PodRun(podSpec *PodSpec) (io.ReadCloser, <-chan int, error)
 	CreateOrUpdateDeploy(deploySpec *DeploySpec) error
-	HasService(namespace, name string) (bool, error)
-	CreateService(namespace, name string) error
+	ExposeApp(namespace, name, vHost string, w io.Writer) error
```

Comment:
I know it's a bit late, but, what about `ExposeDeploy`? `K8sOperations` care about k8s stuff and `app` terminology is something related to Teresa

Reply:
Make sense

Reply:
🔁 , I change `deployOps.exposeService` to `deployOps.exposeApp` and `k8sOps.ExposeApp` to `k8sOps.ExposeDeploy`. c/c @aguerra

Reply:
Great